OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -109,6 +112,9 @@ public override void DoImpact(Target target, WarheadArgs args)
 			var explosion = Explosions.RandomOrDefault(world.LocalRandom);
 			if (Image != null && explosion != null)
 			{
+				if (Inaccuracy.Length > 0)
+					pos += WVec.FromPDF(world.SharedRandom, 2) * Inaccuracy.Length / 1024;
```

Comment:
Why isn't this using the same logic we use in all other places for inaccuracy?

Reply:
I got this from `Bullet`, though...?

Reply:
This bullet? :P https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Projectiles/Bullet.cs#L151-L155

Reply:
Implementing the other inaccuracy types on a warhead would require keeping track of the distance between source and target (including range modifiers and such), which opens a can of worms in terms of what args need to be added/passed from `ProjectileArgs` on to `WarheadArgs`.
It's certainly worth thinking about, but I don't feel like dealing with that in the short-term (iow, before next release).